js/main.js

```diff
@@ -59,6 +89,30 @@ function renderEntry(entry) {
 }
 
 const $ul = document.querySelector('ul');
+$ul.addEventListener('click', clickPencil);
+
+const $title = document.querySelector('#title');
+const $notes = document.querySelector('#notes');
+const $h1 = document.querySelector('h1');
+
+function clickPencil(event) {
+  if (event.target.getAttribute('class') === 'fa fa-pencil fa-lg') {
+    viewSwap('entry-form');
```

Comment:
You should call `viewSwap` after all your logic. Swapping views is the last thing we want to do